lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -1696,7 +1696,7 @@ static ConstString unwrapSILPackType(ConstString mangledName,
 bool SwiftLanguageRuntimeImpl::GetDynamicTypeAndAddress_Pack(
     ValueObject &in_value, CompilerType pack_type,
     lldb::DynamicValueType use_dynamic, TypeAndOrName &pack_type_or_name,
-    Address &address) {
+    Address &address, Value::ValueType &value_type) {
```

Comment:
Why do we need both — doesn't the address cary its own type?

Reply:
This is just threading the `Value::ValueType` that `SwiftLanguageRuntimeImpl::GetDynamicTypeAndAddress` is supposed to fill in to all the `SwiftLanguageRuntimeImpl::GetDynamicTypeAndAddress_*` variants.

> doesn't the address cary its own type?

I think it'd be good if it did, but it doesn't.